### DISCUSSION TITLE
prisma Accelerate

### DISCUSSION THREAD

**Discussion by R0N1n-dev** [prisma/prisma#21797] [Q&A]
### Question

Does prisma accelerate work with findRaw queries

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @R0N1n-dev 👋 

Thank you for raising this question.

It's possible to use `findRaw`or any raw queries with Accelerate, but do note that the responses will not be cached as mentioned in the [documentation](https://www.prisma.io/docs/data-platform/accelerate/limitations#cannot-cache-raw-queries)

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by R0N1n-dev** [comment]
OK.
